### DISCUSSION TITLE
Rebuild with ICU 70.1

### DISCUSSION THREAD

**Discussion by shivammathur** [shivammathur/homebrew-php#805] [Announcements]
Homebrew has updated the `icu4c` formula to `70.1` and subsequently all `php` formulae in this tap have been rebuilt with it.

To upgrade, please reinstall `icu4c` and the `php` formula for the version you use.

For example to upgrade php 8.0:
```
brew update
brew reinstall icu4c shivammathur/php/php@8.0
```

Restart your terminal and verify the PHP version and ICU version

```
php -v
php -i | grep "ICU version"
```

At last, [restart your webserver](https://github.com/shivammathur/homebrew-php#restart-your-webserver).

**Comment by Eseperio** [comment]
If you run `brew reinstall icu4c` and `brew reinstall shivammathur/php/php@7.3` it does not work. I only was able to fix this by calling both reinstallations within the same command.
